src/viam/ur/module/test.cpp

```diff
@@ -77,17 +77,17 @@ BOOST_AUTO_TEST_CASE(test_write_trajectory_to_file) {
 }
 
 BOOST_AUTO_TEST_CASE(test_waypoints_filename) {
-    auto x = waypoints_filename("/home/user", 1747161493357);
+    auto x = waypoints_filename("/home/user", std::chrono::milliseconds(1747161493357));
```

Comment:
Cool trick: use the literals: https://en.cppreference.com/w/cpp/chrono/operator%2522%2522ms.html

```
using namespace std::chrono_literals;
auto x = waypoints_filename("...", 1747161493357us)
```